tool/tctl/common/plugin/awsic_test.go

```diff
@@ -0,0 +1,185 @@
+// Teleport
+// Copyright (C) 2025 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package plugin
+
+import (
+	"math/rand/v2"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/gravitational/teleport/api/types"
+)
+
+func TestAWSICUserFilters(t *testing.T) {
+	t.Run("parse", func(t *testing.T) {
+		// GIVEN some arbitrary set of account name filter CLI args
+		cliArgs := awsICArgs{
+			userLabels: map[string]string{"a": "alpha", "b": "bravo", "c": "charlie"},
+		}
+
+		// WHEN I attempt to convert the command-line args into filters
+		actualFilters, err := cliArgs.parseUserFilters()
+
+		// EXPECT the operation to succeed
+		require.NoError(t, err)
```

Comment:
nit: the comment is really redundant for `require.(No)Error`

Reply:
I prefer to call out each assertion specifically. In a lot of our tests I get lost in teasing apart what is set up, what is the action to test and what the actual assertions are, so I like to be *blindingly* obvious in my own tests. Yeah, it's overkill here - but the practice keeps me honest in more complicated cases.

Reply:
I've at least bundled the two assertions under one comment, if that is better.